source/manual/datalabs-bigquery.html.md

```diff
@@ -0,0 +1,86 @@
+- `ga_sessions_intraday_YYYYMMDD` is the table name
+- `YYYYMMDD` is the current date in year-month-day format
+
+At the end of the day, UA automatically moves the data in this intraday table to a `ga_sessions_YYYYMMDD` table, and creates a new intraday table for the next day.
+
+If there are any issues with this process, you should contact a Google BigQuery administrator. The administrator will raise a request with Merkle, the agency that manages our Google Analytics / BigQuery relationship to fix the issue. Ask in the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) who the current admins are.
+
+The schema of these tables is defined by the [UA BigQuery Export schema](https://support.google.com/analytics/answer/3437719?hl=en). However, not all columns are filled in our tables.
+
+See the [GOV.UK Analytics page on Confluence](https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/23855552/Analytics+on+GOV.UK#AnalyticsonGOV.UK-customDimensionsCustomdimensions) for definitions of our custom dimensions.
+
+There are many other datasets and tables on Google BigQuery. Each project usually has its own dataset.
+
+See the [example queries page](LINK) for more information.
+
+## Get access to Google BigQuery
+
+To get access to Google BigQuery, ask the Data Labs team admins to set you up with a service account and access to the Google BigQuery console. Ask in the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) who the current admins are.
+
+Once you have a service account, store your account credentials safely. For example, store the credentials in your home directory if you have a Unix machine.
+
+Access the [Google BigQuery console](http://console.cloud.google.com/bigquery?organizationId=262359340232&project=govuk-bigquery-analytics&ws=). You can use the console to:
+
+- view projects, datasets, and tables
+- write and manage queries in the editor
+- run those queries
+
+See the [Google BigQuery documentation](https://cloud.google.com/bigquery/docs/how-to) for more information on how to use BigQuery.
+
+## Google BigQuery best practice
+
+When using Google BigQuery you should:
+
+- minimise costs
+- optimise performance
+
+### Minimise Google BigQuery costs
+
+Google BigQuery charges based on the amount of data queried. You should try to minimise the amount of data you query to minimise costs.
+
+You should not use `SELECT *` in your queries if possible, as you could potentially be querying many unnecessary columns of data. Also, you should make sure you do not [query too many tables when using wildcards](https://cloud.google.com/bigquery/docs/querying-wildcard-tables).
+
+Use the query validator to the bottom right of the editor in the console to check query costs before running that query. For API queries, for example using Python, consider [using a dry run step to validate the query costs before running the code](https://cloud.google.com/bigquery/docs/best-practices-costs#performing_a_dry_run).
```

Comment:
a more precise link for the python dry run is https://cloud.google.com/bigquery/docs/best-practices-costs#python